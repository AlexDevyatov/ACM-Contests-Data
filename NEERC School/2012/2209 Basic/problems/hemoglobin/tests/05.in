10
+21
-
+45
-
+4
+28
-
?1
?1
?1
