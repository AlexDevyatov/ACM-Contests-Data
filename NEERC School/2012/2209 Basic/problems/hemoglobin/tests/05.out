21
45
28
4
4
4
